tooling/debugger/ignored-tests.txt

```diff
@@ -12,3 +12,5 @@ references
 scalar_mul
 signed_comparison
 to_bytes_integration
+fold_basic
```

Comment:
@TomAFrench you were correct about having to ignore these, we must not be running these tests on the aztec-packages CI.